messages/device-added-notification.json

```diff
@@ -38,7 +38,419 @@
+                          "type": "string",
+                          "title": "Input type",
+                          "description": "The type of the input",
+                          "enum": [
+                            "object"
+                          ]
+                        },
+                        "properties": {
+                          "type": "object",
+                          "title": "Action input properties",
+                          "description": "A map of action input properties",
+                          "patternProperties": {
+                            ".*": {
+                              "type": "object",
+                              "title": "Action input property",
+                              "description": "An action input property",
+                              "properties": {
+                                "@type": {
+                                  "type": "string",
+                                  "title": "Input property type",
+                                  "description": "The type of the input property"
+                                },
+                                "type": {
+                                  "type": "string",
+                                  "title": "Input property value type",
+                                  "description": "The type of the input property value",
+                                  "enum": [
+                                    "null",
+                                    "boolean",
+                                    "object",
+                                    "array",
+                                    "number",
+                                    "integer",
+                                    "string"
+                                  ]
+                                },
+                                "unit": {
+                                  "type": "string",
+                                  "title": "Input property unit",
+                                  "description": "The unit of the input property"
+                                },
+                                "minimum": {
+                                  "type": "number",
+                                  "title": "Input property minimum value",
+                                  "description": "The minimum value of the input property"
+                                },
+                                "maximum": {
+                                  "type": "number",
+                                  "title": "Input property maximum value",
+                                  "description": "The maximum value of the input property"
+                                },
+                                "multipleOf": {
+                                  "type": "number",
+                                  "title": "Input property value precision",
+                                  "description": "The precision of the value"
```

Comment:
`enum` could be added for action inputs, in addition to `minimum`, `maximum`, and `multipleOf`.

Reply:
Would it make sense to also add it to the spec?

Reply:
I also added it to events.